src/main/resources/GoogleApps/JSExtension.xml

```diff
@@ -20,26 +20,29 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.5" reference="GoogleApps.JSExtension" locale="">
   <web>GoogleApps</web>
   <name>JSExtension</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
-  <parent>GoogleApps.WebHome</parent>
   <creator>xwiki:XWiki.Admin</creator>
+  <parent>GoogleApps.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
-  <customClass/>
+  <originalMetadataAuthor>XWiki.superadmin</originalMetadataAuthor>
```

Comment:
This should have been removed by the ``xar:format``, normally. I'm surprised it was kept. We don't keep it on XS pages. Would be good to remove it manually if ``xar:format`` keeps it.